app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template">
+  <head>
+    <title><%= [yield(:page_title).presence, t('service.name')].compact.join(' - ') %></title>
+
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= tag :meta, name: 'viewport', content: 'width=device-width, initial-scale=1' %>
```

Comment:
there is no images directory containing these assets, so the page load fails.

Reply:
it doesn't? There's a test that shows the page load working